docs/3.x.x/getting-started/quick-start.md

```diff
@@ -1,285 +1,281 @@
-  })
-  .catch(error => {
-    // Handle error.
-    console.log('An error occurred:', error);
-  });
-```
+1. We are going to give **Public** (any web browser with the correct link) access to the **Restaurant** Content Type.
 
-### Get a specific entry (GET)
+   Locate and click on the **Roles & Permission** menu item under **PLUGINS** on the left menu.
 
-If you want to get a specific entry, add the `id` of the wanted post at the end of the url.
+   (The **Roles & Permission** plugin can accomplish many tasks related to permissions. For now we will focus on the **Public** role.)
 
-Example with Axios:
+   Next, click on the **pencil** edit icon to the right of the **Public** Role.
 
-```js
-import axios from 'axios';
+   From here, scroll down under **Permissions** and find **Restaurant**.
 
-const postId = 'YOUR_POST_ID_HERE'; // Replace with one of your posts id.
+   Click the checkbox next to **find**. To the right, you will see the URL route. It should say, `/restaurants"`.
 
-// Request API.
-axios
-  .get(`http://localhost:1337/posts/${postId}`)
-  .then(response => {
-    // Handle success.
-    console.log('Well done, here is the post: ', response.data);
-  })
-  .catch(error => {
-    // Handle error.
-    console.log('An error occurred:', error);
-  });
-```
+   Scroll back to the top, and click the **Save** button.
 
-### Create data (POST)
-
-Use the `POST` route to create a new entry.
-
-Example with Axios:
-
-```js
-import axios from 'axios';
-
-// Request API.
-axios
-  .post(`http://localhost:1337/posts/`, {
-    title: 'My new post'
-  })
-  .then(response => {
-    // Handle success.
-    console.log(
-      'Well done, your post has been successfully created: ',
-      response.data
-    );
-  })
-  .catch(error => {
-    // Handle error.
-    console.log('An error occurred:', error);
-  });
-```
+2. You are returned to the **Roles and Permission** Panel.
 
-### Update data (PUT)
-
-Use the `PUT` route to update an existing entry.
-
-Example with Axios:
-
-```js
-import axios from 'axios';
-
-const postId = 'YOUR_POST_ID_HERE'; // Replace with one of your posts id.
-
-// Request API.
-axios
-  .put(`http://localhost:1337/posts/${postId}`, {
-    title: 'Updated title'
-  })
-  .then(response => {
-    // Handle success.
-    console.log(
-      'Well done, your post has been successfully updated: ',
-      response.data
-    );
-  })
-  .catch(error => {
-    // Handle error.
-    console.log('An error occurred:', error);
-  });
-```
+We are now ready to [Consume the Content Type API](#_6-consume-the-content-type-api).
 
-### Delete data (DELETE)
+Click on the Strapi Logo to return to the main administration panel.
 
-Use the `DELETE` route to delete an existing entry.
+---
 
-Example with Axios:
+## 6. Consume the Content Type API
 
-```js
-import axios from 'axios';
+The Project is accessible by following the `http://localhost:1337/` link. You will see the **'Welcome'** screen.
```

Comment:
Add a gif of Consuming the API https://drive.google.com/open?id=19h91Z5hUXTzsEhyZfhFiS9EQVSlorlGc

Reply:
n/A we now have the videos for each section.